integrations/ultralytics/server/server.py

```diff
@@ -0,0 +1,267 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Example script for hosting a Yolo ONNX model as a Flask server
+using the DeepSparse Engine as the inference backend
+
+##########
+Command help:
+usage: server.py [-h] [-s BATCH_SIZE] [-j NUM_CORES] [-a ADDRESS] [-p PORT]
+ onnx_filepath
+
+Host a Yolo ONNX model as a server, using the DeepSparse Engine and Flask
+
+positional arguments:
+  onnx_filepath         The full filepath of the ONNX model file or SparseZoo stub
+                        to the model
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -s BATCH_SIZE, --batch_size BATCH_SIZE
+                        The batch size to run the analysis for
+  -j NUM_CORES, --num_cores NUM_CORES
+                        The number of physical cores to run the analysis on,
+                        defaults to all physical cores available on the system
+  -a ADDRESS, --address ADDRESS
+                        The IP address of the hosted model
+  -p PORT, --port PORT  The port that the model is hosted on
+
+##########
+Example command for running:
+python server.py \
+    ~/models/yolo-v3-pruned_quant.onnx
+"""
+
+import argparse
+import time
+from typing import List, Tuple
+
+import numpy
+import torch
+
+import cv2
+import flask
+from deepsparse import compile_model
+from flask_cors import CORS
+
+# ultralytics/yolov5 imports
+from utils.general import non_max_suppression
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description=(
+            "Host a Yolo ONNX model as a server, using the DeepSparse Engine and Flask"
+        )
+    )
+
+    parser.add_argument(
+        "onnx_filepath",
```

Comment:
for our standard args in other scripts, we use "-" as the separator, can we update for consistency along with the other params?